pkg/reconciler/v1alpha1/clusteringress/resources/virtual_service.go

```diff
@@ -77,7 +77,7 @@ func makeVirtualServiceSpec(ci *v1alpha1.ClusterIngress, gateways []string) *v1a
 	for _, rule := range ci.Spec.Rules {
 		hosts := rule.Hosts
 		for _, p := range rule.HTTP.Paths {
-			spec.Http = append(spec.Http, *makeVirtualServiceRoute(hosts, &p))
+			spec.HTTP = append(spec.HTTP, *makeVirtualServiceRoute(hosts, &p))
```

Comment:
https://github.com/knative/serving/pull/3590 handles just this rename and should be in soon.

Reply:
ack